package-overrides/github/CodeSeven/toastr@2.1.1.json

```diff
@@ -0,0 +1,22 @@
+{
+  "main": "toastr",
+  "registry": "github",
+  "format": "global",
+  "dependencies": {
+    "jquery": "github:components/jquery@2.1.3",
+    "css": "github:systemjs/plugin-css@^0.1.13"
```

Comment:
Best to leave these as the widest ranges possible - `jquery: 'github:components/jquery'` (meaning `*`), and `*` for CSS as well.